crates/storage/libmdbx-rs/src/transaction.rs

```diff
@@ -586,8 +586,9 @@ impl TransactionPtr {
             tracing::debug!(
                 target: "libmdbx",
                 txn = %self.txn as usize,
-                backtrace = %std::backtrace::Backtrace::force_capture(),
-                "Transaction lock is already acquired, blocking..."
+                backtrace = %std::backtrace::Backtrace::capture(),
```

Comment:
this should now just be an atomic load so this is better since this path is now reachable more often